### PR TITLE
ENH: Add bokeh in-line plotting to the header page

### PR DESCRIPTION
**Pull request by ericdill**
cc @danielballan

### CODE REVIEW

**Comment by danielballan** [comment]
accidentally committed nbs

File: album/server.py [hunk 1]
```diff
@@ -25,14 +39,59 @@ def run_index():
 
 
 @app.route('/run/<uid>')
-def run_show(uid):
+def run_show(*args, **kwargs):
```
Comment (danielballan) [inline album/server.py]: Why?

**Comment by ericdill** [comment]
"ugh i need to remove the ipython notebook"

was the comment that I was about to add when @arkilic came by

**Comment by danielballan** [comment]
Hahha cool

File: album/server.py (hunk 1 above)
Comment (ericdill) [inline album/server.py]: long story short I was trying to debug bokeh oddities. This needs to be reverted

**Comment by danielballan** [comment]
Can we put the bokeh-specific imports and plotting code in a separate module, to isolate the dep and keep `server.py` close to an MVP?

**Comment by ghost** [comment]
@ericdill thx for letting gh know I stopped by

**Comment by ericdill** [comment]
yw

**Comment by ericdill** [comment]
Rebased to get rid of a number of no-longer-needed files

File: album/bokeh_plot.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  {{ plot_script|indent(4)|safe }}
+  </head>
+  <body>{{ plot_div|indent(4)|safe }}</body>"""
+
+def plot_table_by_time(table):
+    plots = []
+    for k, v in table.items():
+        if k == 'time':
+            continue
+        df = pd.DataFrame(v.values, index=table['time'], columns=[k])
```
Comment (danielballan) [inline album/bokeh_plot.py]: What about the Bokeh df-plotting widget, then?

**Pull request merged**